cdap-ui/app/directives/complex-schema/complex-schema.less

```diff
@@ -75,16 +75,16 @@ my-complex-schema {
 
       /**
        * 75px is the width of the checkbox, trash, and add button
-       * 80px is the width of the type dropdown
+       * 105px is the width of the type dropdown
        **/
-      width: ~"-moz-calc(100% - 75px - 80px)";
-      width: ~"-webkit-calc(100% - 75px - 80px)";
-      width: ~"calc(100% - 75px - 80px)";
+      width: ~"-moz-calc(100% - 75px - 105px)";
```

Comment:
nit: Since we define a variable `@type-column-width` can we use that in occurrences of `105px`?